pipeline/merger/src/main/scala/uk/ac/wellcome/platform/merger/models/FieldMergeResult.scala

```diff
@@ -9,4 +9,4 @@ import uk.ac.wellcome.models.work.internal.TransformedBaseWork
  * the merger how to handle these.
  */
 case class FieldMergeResult[T](fieldData: T,
-                               redirects: Seq[TransformedBaseWork])
+                               mergedFrom: Seq[TransformedBaseWork])
```

Comment:
I'm not sure about `mergedFrom` as a name, I don't think it makes the most sense when it's used outside of these rules. I think a plural of a noun is a less confusing way of naming this, and it might make sense to be explicit about the fact that these are a subset of the sources, eg:

- `mergedSources`
- `usedSources`
- `mergedFromSources`
- `dataSources`